src/components/Nav/Nav.js

```diff
@@ -1,13 +1,42 @@
-import React from 'react';
-import styled from 'styled-components/macro';
+import React, { useState, useEffect } from 'react';
+import styled, { css } from 'styled-components/macro';
 import { Link, useLocation } from 'react-router-dom';
+import { API } from '../../config';
 
 function Nav() {
   const location = useLocation();
+  const main = location.pathname === '/';
+  const me = location.pathname === '/me';
+
+  const [user, setUser] = useState({ userInfo: '' });
+  const { userInfo } = user;
```

Comment:
userInfo 외에 관련된 부가 정보가 함께 관리되어야 하는 경우가 아닌데, 객체의 형태로 관리하는 이유가 있으신가요?
```suggestion
  const [userInfo, setUserInfo] = useState('');
```

Reply:
아뇨! useState를 사용하는 게 익숙하지 않아서 저렇게 작성한 것 같습니다! 초기값을 빈 스트링으로 수정하겠습니다!